### DISCUSSION TITLE
New to cloud dev, want to start with GitHub Action & Deploy on Azure

### DISCUSSION THREAD

**Discussion by ghost** [community/events#30] [Cloud]
Thanks for the CodeSpaces announcement, please leave a comment here if you want to.

**Comment by ossanna16** [comment]
Hi there @Robertblazor! Thank you for your post! Is there a specific question we can help you with?

**Reply by ghost** [reply]
I want to learn GitHub Actions with Azure Kubernetes, starting with the repo here:

https://github.com/dotnet-architecture/eShopOnContainers

How do I deploy it on Azure and make my own workflow?

**Reply by ossanna16** [reply]
Perhaps [this](https://learn.microsoft.com/en-us/azure/aks/kubernetes-action) may be helpful to you.

**Reply by ghost** [reply]
Thanks a lot, this is excellently helpful.

**Reply by ossanna16** [reply]
You're welcome 🙂
